internal/lib/config/config.go

```diff
@@ -140,9 +140,10 @@ type RootConfig struct {
 // RuntimeHandler represents each item of the "crio.runtime.runtimes" TOML
 // config table.
 type RuntimeHandler struct {
-	RuntimePath string `toml:"runtime_path"`
-	RuntimeType string `toml:"runtime_type"`
-	RuntimeRoot string `toml:"runtime_root"`
+	RuntimePath                  string `toml:"runtime_path"`
+	RuntimeType                  string `toml:"runtime_type"`
+	RuntimeRoot                  string `toml:"runtime_root"`
+	PrivilegedWithoutHostDevices bool   `toml:"privileged_without_host_devices"`
```

Comment:
Can we please add documentation about this flag? I'm also not sure if the naming is that concrete. What do you think about something like `mount_host_devices_on_privileged_containers`?